bundles/InstallBundle/Command/InstallCommand.php

```diff
@@ -189,9 +190,15 @@ protected function configure()
     protected function initialize(InputInterface $input, OutputInterface $output)
     {
         // no installer if Pimcore is already installed
-        $configFile = Config::locateConfigFile('system.yml');
-        if ($configFile && is_file($configFile) && !$input->getOption('ignore-existing-config')) {
-            throw new \RuntimeException(sprintf('The system.yml config file already exists in "%s". You can run this command with the --ignore-existing-config flag to ignore this error.', $configFile));
+        $configFile = Config::locateConfigFile('parameters.yml');
```

Comment:
not necessary, all that information is in the container and can be loaded from there

Reply:
this returns parameters.yml file path to check if Pimcore is already installed and prompt user to provide --ignore-existing-config to proceed.

Reply:
but the parameters.yml is copied from Pimcore\Composer before an installation actually happened.

Reply:
this "/app/config/local/parameters.yml" is different from /app/config/parameters.yml

Reply:
puh.... not sure if thats better.

Is this test even necessary? If it is: Why not check the DB if a certain table exists?

Reply:
How should we check if a certain table exists without being able to establish a DB connection (because Pimcore is not installed) ... sounds like an interesting approach 🤔